src/static/registry.json

```diff
@@ -1008,7 +1008,7 @@
                 "testing": "Partial",
                 "devstatus": "Good",
                 "python3": "Yes",
-                "last-updated": "2019-11-04"
+                "last-updated": "2022-02-04"
```

Comment:
This is pulled from https://github.com/astropy/astropy.github.com/blob/298c6b20f393f3dad8794e511fef685c1fed5dd1/affiliated/registry.json#L996 . If you want to change this, you need to update it first over there.

Also, you don't have to touch `last-updated` as that field is reserved for re-reviewing the package.

Reply:
@pllim  Ok, I just reset the original "last-updated" value.